### PR TITLE
EFX-205 add links for stats cards

### PR DESCRIPTION
**Pull request by ainursharaev**
This MR adds links to cards, so users will be redirected to corresponding web pages on clicking a card.

The following links were chosen:

- size – https://bundlephobia.com/package/effector@latest
- contributors – https://github.com/effector/effector/graphs/contributors
- stars – https://star-history.com/#effector/effector&Date
- downloads – https://npmtrends.com/effector
- latest – https://github.com/effector/effector/releases
- license – https://raw.githubusercontent.com/effector/effector/master/LICENSE

Looking forward to hearing your opinions and suggestions 👍

### CODE REVIEW

File: beta/src/components/Landing/Stats.astro [hunk 1]
```diff
@@ -96,33 +96,53 @@ async function getGzipSize() {
...
     <div class="counts">
       {
-        Object.entries(stats).map(([label, counter]) => {
+        Object.entries(stats).map(([name, counter]) => {
           return (
             <div class="stats-card">
               <div class="text-4xl">{counter}</div>
-              <div>{labelMapper[label]}</div>
+              <div>{statsMeta[name].label}</div>
+              <a href={statsMeta[name].link} />
```
Comment (sergeysova) [inline beta/src/components/Landing/Stats.astro:145]:
```suggestion
              <a href={statsMeta[name].link} target="_blank" rel="noreferrer noopener" />
```

Comment (sergeysova) [reply beta/src/components/Landing/Stats.astro:145]: Why not the whole block wrap in the `a`?

Comment (sergeysova) [reply beta/src/components/Landing/Stats.astro:145]:
```astro
<a href="" target="_blank" rel="..." class="stats-card">
  <div class="text-4xl">{counter}</div>
  <div>{statsMeta[name].label}</div>
  <a href={statsMeta[name].link} />
```

File: beta/src/components/Landing/Stats.astro [hunk 2]
```diff
@@ -145,53 +165,57 @@ const stats = await Promise.all([getRepoStats(), getNpmStats(), getGzipSize()]).
 </div>
 
 <style>
-  .wrapper {
-    @apply mx-auto max-w-6xl p-3 sm:p-6;
-  }
+    .wrapper {
+        @apply mx-auto max-w-6xl p-3 sm:p-6;
```
Comment (sergeysova) [inline beta/src/components/Landing/Stats.astro:169]: Really, prettier loaded wrong config 😢

**Review by sergeysova — CHANGES_REQUESTED** [review]

File: beta/src/components/Landing/Stats.astro [hunk 3]
```diff
@@ -44,7 +44,7 @@ async function getNpmStats() {
 
   return {
     latest,
-    downloads: `${totalDownloads.toFixed(0)}K`,
+    downloads: `${totalDownloads.toFixed(0)}K`
```
[the remark this answers is not in the document]
Comment (ainursharaev) [reply beta/src/components/Landing/Stats.astro]: Used the correct config in IDE 👍  thanks for bringing this to my attertion

File: beta/src/components/Landing/Stats.astro (hunk 1 above)
Comment (ainursharaev) [reply beta/src/components/Landing/Stats.astro]: Hmm, makes sense 👍 I edited this part

File: beta/src/components/Landing/Stats.astro (hunk 2 above)
Comment (ainursharaev) [reply beta/src/components/Landing/Stats.astro]: I've changed my IDE configuration, thanks 👍

**Review by sergeysova — APPROVED** [review]

**Pull request merged**